### PR TITLE
Li7200

### PR DESCRIPTION
**Pull request by liangyingz**

### CODE REVIEW

File: pipe/li7200/li7200_assetuid_macaddress_map.yaml
Comment (covesturtevant) [inline pipe/li7200/li7200_assetuid_macaddress_map.yaml:1]: With the removal of the pod_patch portion, I think you'll want to add sidecar_resource_requests. See nearly any other pipeline spec for an example.

File: pipe/turbulent/turbulent_group_loader.yaml
Comment (covesturtevant) [inline pipe/turbulent/turbulent_group_loader.yaml:1]: same here. You'll probably want sidecar_resource_requests for stability.

**Review by covesturtevant — APPROVED** [review]
Looks good. See the 2 comments about sidecar_resource_requests. If/when these pipelines are relatively stable, you might also want to stand up github action workflows to get automatic updates when image versions get updated, etc.

**Comment by liangyingz** [comment]
Thank you Cove! I will add sidecar_resource_requests to those two pipelines. Also work on github action workflow. Will merge those changes together after done.

**Pull request merged**